### PR TITLE
Add option to show or hide close icon from notifications

### PR DESCRIPTION
**Pull request by kpunith8**

### CODE REVIEW

**Comment by MohammedRajaAtLookout** [comment]
I was not able to create a PR from my machine. It shows a permission issue, maybe I messed with the ssh keys.
It is just a small change and requested Punith to create a PR.
Thank Punith!

File: views/notification.hbs [hunk 1]
```diff
@@ -1,4 +1,6 @@
 <span class="notification-message">{{message}}</span>
-<button type="button" class="close" data-dismiss="alert" aria-hidden="true">
-  {{icon 'misc-x'}}
-</button>
+{{#if showCloseButton}}
```
Comment (eugenet8k) [inline views/notification.hbs:2]: To make this work, you need to update all existing code and pass `showCloseButton: true`. And only for the case where you need to hide the button you would be able to skip this parameter. The requirement is to do quite opposite, having a flag that allows removing the close button from notification if it's not needed, where by default it should stay visible.

File: views/notification-view.spec.coffee [hunk 2]
```diff
@@ -9,13 +9,14 @@ describe 'NotificationView', ->
...
   useFakeTimers = yes
+  showCloseButton = true
 
   beforeEach ->
     sandbox = sinon.createSandbox {useFakeTimers}
     success = sinon.spy()
     undo = sinon.spy()
     opts = {success, undo, model: new Chaplin.Model()}
-    model = new Chaplin.Model {opts}
+    model = new Chaplin.Model {opts, showCloseButton}
```
Comment (eugenet8k) [inline views/notification-view.spec.coffee:19]: the newly adding parameter should be part of `opts` object. "opts" name stand for options and this is yet another option.

**Review by eugenet8k — CHANGES_REQUESTED** [review]

**Comment by eugenet8k** [comment]
Yeh, if you had a custom config in your «~/.npmrc» for Lookout artifactory that might affect regular GitHub access.

Unfortunately the code in your PR is wrong.

**Comment by eugenet8k** [comment]
Looks good thank you

**Review by eugenet8k — CHANGES_REQUESTED** [review]
There should be one commit in this PR.

**Comment by kpunith8** [comment]
> Please squash two commits into one before merging.

I squashed into a single commit

**Pull request merged**